overlord/servicestate/quota_control_test.go

```diff
@@ -921,3 +938,226 @@ func (s *quotaControlSuite) TestCreateQuotaCreateQuotaConflict(c *C) {
 	_, err = servicestate.CreateQuota(st, "foo", "", []string{"test-snap2"}, 2*quantity.SizeGiB)
 	c.Assert(err, ErrorMatches, `quota group "foo" has "quota-control" change in progress`)
 }
+
+func (s *quotaControlSuite) TestMemoryCGroupDisabled(c *C) {
+	s.state.Lock()
+	defer s.state.Unlock()
+
+	_memoryCGroupFile := servicestate.CGroupsFilePath
```

Comment:
super super super nitpick, but we don't typically use the `_` in Go variable names

Reply:
updated the names accordingly